torch/_torch_docs.py

```diff
@@ -2543,17 +2543,22 @@ def merge_dicts(*dicts):
            r"""
 logical_not(input, out=None) -> Tensor
 
-Computes the element-wise logical NOT of the given input tensor. The input tensor must have the bool dtype.
+Computes the element-wise logical NOT of the given input tensor. The output tensor always have the bool dtype. If the
+input tensor is not a bool tensor, zeros are treated as ``False`` and non-zeros are treated as ``True``.
 
 Args:
-    input (Tensor): the input tensor
-    out (Tensor, optional): the output tensor
+    {input}
+    {out}
 
-Example:
+Example::
 
     >>> torch.logical_not(torch.tensor([True, False]))
     tensor([ False,  True])
-""")
+    >>> torch.logical_not(torch.tensor([0, 1, -10], dtype=torch.int8))
+    tensor([ True, False,  False])
+    >>> torch.logical_not(torch.tensor([0, 1, -10], dtype=torch.double))
```

Comment:
nit: this is a bit of a strange example -- we should probably have decimal values.